crates/bevy_ecs/src/world/entity_ref.rs

```diff
@@ -375,6 +375,70 @@ impl<'w> EntityMut<'w> {
         self
     }
 
+    /// Inserts a component with the given `value`. Will replace the value if it already existed.
+    ///
+    /// **You should prefer to use the typed API [`EntityMut::insert`] where possible and only
+    /// use this in cases where there isn't a Rust type corresponding to the [`ComponentId`].
+    ///
+    /// # Safety
+    /// - `value` must be valid for the given [`ComponentId`] of this world
+    /// - storage type must be valid for the given [`ComponentId`] of this world
+    pub unsafe fn insert_by_id(
```

Comment:
This method, and the one below, should link to the init_dynamic_info method from above to explain how to fetch the bundle info.

Reply:
Done